local-cluster/Main.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE NumericUnderscores #-}
+module Main (main) where
+
+import Test.Plutip.LocalCluster
+    ( addSomeWallet,
+      mkMainnetAddress,
+      startCluster,
+      stopCluster,
+      waitSeconds )
+import Data.Default (def)
+import Control.Monad (void)
+import Control.Monad.Reader (ReaderT (ReaderT))
+import Test.Plutip.Internal.Types ( nodeSocket )
+import Control.Monad.IO.Class (liftIO)
+import Test.Plutip.Internal.BotPlutusInterface.Wallet (walletPkh)
+
+main :: IO ()
+main = do
+  (st, _) <- startCluster def $ do
+    w <- addSomeWallet (toAda 10000)
```

Comment:
Hi @mikekeke thanks for doing this, very cool.

Let me illustrate a use case perhaps that would be useful for you. I would like to run something like

```
$ plutip local-cluster run --wallets 10 --wallets-dir $TMPDIR/local-cluster-wallets
```

The frontend would need to pick up these generated wallets to run contracts with. We currently run BPI contracts and CTL contracts in the browser. Does that make sense?

Reply:
I'm actually was using alike setup for a while - some wallets at my disposal and keys for them in known location, for separate instance of bot interface to pick-up them, - but had to hardcode parameters.
Making it not hacky way will take some time, though. Maybe make separate issue out of it? What do you think @gege251?